### PR TITLE
fix: ecocredit module migration

### PR DESCRIPTION
**Pull request by robert-zaremba**
---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] followed the guidelines for [building modules](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules)
- [ ] included the necessary unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed 
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: app/stable_appconfig.go [hunk 1]
```diff
@@ -49,60 +45,23 @@ func setCustomKVStoreKeys() []string {
...
-		gen.Params.AllowlistEnabled = true
-		gen.Params.CreditClassFee = sdk.NewCoins(sdk.NewCoin("uregen", ecocredittypes.DefaultCreditClassFeeTokens))
-
-		modules := make(map[string]json.RawMessage)
-		modules[ecocredittypes.ModuleName] = app.cdc.MustMarshalJSON(gen)
-		app.smm.InitGenesis(ctx, modules, []abci.ValidatorUpdate{})
-
-		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
-	})
+		// set basket creation fee to 1'0000 REGEN
```
Comment (technicallyty) [inline app/stable_appconfig.go:60]:
```suggestion
		// set basket creation fee to 1,000 REGEN
```

**Review by technicallyty — APPROVED** [review]

File: app/stable_appconfig.go (hunk 1 above)
Comment (robert-zaremba) [reply app/stable_appconfig.go:60]: Personally I prefer `'` because it's more clear. In France and Spain, `,` is used for decimal separator.

Comment (technicallyty) [reply app/stable_appconfig.go:60]: maybe just not separate it at all?

**Review by ryanchristo — APPROVED** [review]
tACK

File: app/stable_appconfig.go (hunk 1 above)
Comment (ryanchristo) [reply app/stable_appconfig.go:60]: Let's be consistent and use `,` as we do elsewhere in our codebase.

Comment (ryanchristo) [reply app/stable_appconfig.go:60]: We have been traditionally using `.` for decimals and `,` for long numbers.

**Comment by ryanchristo** [comment]
please update the pull request title before merging

File: app/stable_appconfig.go [hunk 2]
```diff
@@ -6,28 +6,24 @@
...
 	paramsclient "github.com/cosmos/cosmos-sdk/x/params/client"
 	paramskeeper "github.com/cosmos/cosmos-sdk/x/params/keeper"
 	upgradeclient "github.com/cosmos/cosmos-sdk/x/upgrade/client"
 	upgradetypes "github.com/cosmos/cosmos-sdk/x/upgrade/types"
-	abci "github.com/tendermint/tendermint/abci/types"
 
 	"github.com/regen-network/regen-ledger/types/module/server"
-	ecocredittypes "github.com/regen-network/regen-ledger/x/ecocredit"
+	"github.com/regen-network/regen-ledger/x/ecocredit"
+	// ecocreditmodule "github.com/regen-network/regen-ledger/x/ecocredit/module"
```
Comment (ryanchristo) [inline app/stable_appconfig.go:26]:
```suggestion
```

File: app/stable_appconfig.go (hunk 1 above)
Comment (clevinson) [reply app/stable_appconfig.go]: Agreed that `,` is more consistent with our usage elsewhere.

File: app/stable_appconfig.go [hunk 3]
```diff
@@ -49,60 +44,23 @@ func setCustomKVStoreKeys() []string {
 }
 
 func (app *RegenApp) registerUpgradeHandlers() {
+
 	// This is the upgrade plan name we used in the gov proposal.
-	upgradeName := "v2.0-upgrade"
-	app.UpgradeKeeper.SetUpgradeHandler(upgradeName, func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
```
Comment (aleem1314) [inline app/stable_appconfig.go:54]: We can also remove IBC patch [handler](https://github.com/regen-network/regen-ledger/blob/v3.0.0-rc1/app/app.go#L506-L509).

**Review by aleem1314 — APPROVED** [review]

File: app/stable_appconfig.go (hunk 3 above)
Comment (robert-zaremba) [reply app/stable_appconfig.go:54]: true, thanks for note

**Pull request merged**